test_scripts/Defects/5_0/2481_ChoiceID_is_missing_in_PerformInteraction_response.lua

```diff
@@ -0,0 +1,144 @@
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local paramsBoth = {
+    initialText = "Pick an Alphabet",
+    initalPrompt = "Pick a command",
+    interactionMode = "BOTH",
+    interactionChoiceSetIDList = { 1 }
+}
+
+-- [[ Local Functions ]]
+local function setExChoiceSet(choiceIDValues)
+  local exChoiceSet = { }
+  for i = 1, #choiceIDValues do
+    exChoiceSet[i] = {
+      choiceID = choiceIDValues[i]
+    }
+  end
+  return exChoiceSet
+end
+
+local function PromptValue(text)
+  local tmp = {
+    {
+      text = text,
+      type = "TEXT"
+    }
+  }
+  return tmp
+end
+
+local initialPromptValue = PromptValue(" Make your choice ")
+local helpPromptValue = PromptValue(" Help Prompt ")
+local timeoutPromptValue = PromptValue(" Time out ")
+
+local function SendOnSystemContext(pParam)
+  common.getHMIConnection():SendNotification("UI.OnSystemContext",
+    { appID = common.getHMIAppId(), systemContext = pParam })
+end
+
+local function createInteractionChoiceSet()
+    local choiceParams = {
+        interactionChoiceSetID = 1,
+        choiceSet = {
+          {
+            choiceID = 1,
+            menuName = "Choice1",
+            vrCommands = { "Choice1" }
+          }
+        } 
+    }
+    local cid = common.getMobileSession():SendRPC("CreateInteractionChoiceSet", choiceParams)
+    common.getHMIConnection():ExpectRequest("VR.AddCommand")
+    :Do(function(_, data)
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+    common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end 
+
+local function sendPerformInteraction(pParam)
+    local cid = common.getMobileSession():SendRPC("PerformInteraction", pParam)
+    common.getHMIConnection():ExpectRequest("VR.PerformInteraction", {
+      helpPrompt = pParam.helpPrompt,
+      initialPrompt = pParam.initialPrompt,
+      timeout = pParam.timeout,
+      timeoutPrompt = pParam.timeoutPrompt      
+    })
+    :Do(function(_,data)
+      common.getHMIConnection():SendNotification("VR.Started")
+      common.getHMIConnection():SendNotification("TTS.Started")
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+      SendOnSystemContext("VRSESSION")
+      local function firstSpeakTimeOut()
+        common.getHMIConnection():SendNotification("VR.Stopped")
+        common.getHMIConnection():SendNotification("TTS.Stopped")
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+      end
+      RUN_AFTER(firstSpeakTimeOut, 5)
+      local function vrResponse()
+        common.getHMIConnection():SendError(data.id, data.method, "TIMED_OUT", "Perform Interaction error response.")
```

Comment:
third one. Also according to defect description choiceId must be send from VR.PI